### ISSUE TITLE
receipt validation sandbox / production

### ISSUE THREAD

**Issue by bb-git** [bizz84/SwiftyStoreKit#44]
Would it be possible to implement the verification according to apple?

> How do I verify my receipt (iOS)?
> Always verify your receipt first with the production URL; proceed to verify
> with the sandbox URL if you receive a 21007 status code. Following this
> approach ensures that you do not have to switch between URLs while your
> application is being tested or reviewed in the sandbox or is live in the
> App Store.
> 
> Note: The 21007 status code indicates that this receipt is a sandbox receipt,
> but it was sent to the production service for verification.

**Comment by bizz84** [comment]
Thanks for reporting this - unfortunately I haven't had the chance to look into this yet - pull requests would be most welcome.

**Comment by bb-git** [comment]
I'm checking this in the receipt validation method

```
            case .Error(let error):
                if case .ReceiptInvalid(let receiptError) = error{
                    print ("receipt error: \(receiptError)")
                    if case .TestReceipt = receiptError.status{
                        print ("receipt error status: \(receiptError.status)")
                        self.salesVerifyAllPurchases(productId, isProduction: false)
                    }
                }

                if case .NoReceiptData = error {
                    self.salesRefreshReceipt()
                }

            }
```

**Comment by bizz84** [comment]
@bb-git Could you test this pull request and see if it works for you?

https://github.com/bizz84/SwiftyStoreKit/pull/56

**Comment by bizz84** [comment]
Fixed on pod version 0.3.0. Feel free to reopen if there are any issues.
